app/assets/javascripts/test/backend-snapshot-tests/teamstructure.e2e.js

```diff
@@ -0,0 +1,170 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
+/* eslint-disable import/first */
+// @flow
+import {
+  tokenUserDefault,
+  tokenUserAAB,
+  tokenUserAAC,
+  tokenUserABA,
+  tokenUserBAA,
+  setCurrToken,
+} from "../enzyme/e2e-setup";
+import test from "ava";
+import _ from "lodash";
+import * as api from "admin/admin_rest_api";
+
+/*
+TEAM STRUCTURE USED FOR TESTING:
+
+Organizations:
+Connectomics department (Organization A), OrganizationB
+
+Teams:
+Connectomics department (Team AA), Team AB, Team BA, test1, test2
+(the teams test1 and test2 existed before)
+
+User:
+default (UserAAA), User AAB, User AAC, User ABA, User BAA
+
+++----------------------------------------------------------------------------------------------------------------------++--------------------------------------++
+||	Connectomics department (Organization A)																			                                      ||	Organization B 						          ||
+||  Admin: scmboy																										                                                    ||  Admin: userBAA					          	||
+++-----------------------------------+--------------------------+---------------------------+---------------------------++--------------------------------------++
+|| Connectomics department (Team AA) | 		Team AB				        | 		test1				          | 		test2				          ||  TeamBA 								              ||
+++-----------------------------------+--------------------------+---------------------------+---------------------------++--------------------------------------++
+|| scmboy (userAAA)	(teamMng)		     | 		userABA (teamMng.)    | 		scmboy            	  | 		scmboy (teamMng.)	    ||  userBAA	(teamMng.)			         		||
+|| userAAB (teamMng.)		   		       | 						            	| 					              	| 						            	||  							                  		||
+|| userAAC					   		           | 						            	| 					            		| 						            	||  							                  		||
+++-----------------------------------+--------------------------+---------------------------+---------------------------++--------------------------------------++
```

Comment:
Where is this table formatting coming from? In a mono-space font it's not aligned properly. Can I view it somewhere correctly?

Reply:
Is there a pattern between the naming of organization team and user? It's a bit confusing to only use `A` and `B` for all three entity types. You could use different letters like: Organization X, Organization Y, Team 1, Team 2, Team 3 and User A, User B, User C.
If you want to encode the relationship, maybe the following
Organization X, Organization Y, Team X1, Team X2, Team Y3 and User A_X12, User B_Y3, User C_X1.